sequencer/status.go

```diff
@@ -59,7 +59,7 @@ type HubResponse struct {
 	} `json:"stateInfo"`
 }
 
-func getHubHeight(cfg config.RollappConfig) (string, error) {
+func GetLocalRollappHubHeight(cfg config.RollappConfig) (string, error) {
 	cmd := exec.Command(consts.Executables.Dymension, "q", "rollapp", "state", cfg.RollappID,
```

Comment:
It's bad to execute the binary each cycle to update the metrics
bad performance

It's need to be refactored to single Hub RPC client... the current implementation is not scalable

Reply:
it's different
#150 is about REST API vs RPC

here u actually executing a binary